coffee_lib/src/types/mod.rs

```diff
@@ -138,6 +138,7 @@ pub mod response {
         // A patch operation when a git repository is present in the coffee configuration
         // but is absent from the local storage.
         RepositoryLocallyAbsent(Vec<String>),
+        CoffeeGlobalrepoCleanup(Vec<(String, String)>),
```

Comment:
It's not clear what `Vec<(String, String)>` represents
we can add a doc comment here
```suggestion
        /// Affected network, path
        CoffeeGlobalrepoCleanup(Vec<(String, String)>),
```